wiki/osu!_wiki/Contribution_guide/Best_practices/de.md

```diff
@@ -35,27 +30,17 @@ Wenn du während der Verwendung des GitHub-Tools auf irgendwelche Probleme stö
 
 1. Öffne **deine Verzweigung** und gehe zum Tab `Actions`.
 
-2. Schaue unter `Workflows` nach `Sync from osu! upstream`.
+2. Schaue auf der linken Seitenleiste nach `Sync with ppy:master`.
 
 3. Klicke auf `Run workflow` und fülle die Optionen aus:
 
-   <!-- when updating this translation, delete img/github-actions-workflow-dialog.png if this translation is the last one to use it. —clayton -->
-
-   ![](img/github-actions-workflow-dialog.png "GitHub Actions Workflow - Run Workflow")
+   ![Screenshot des Formulars auf der GitHub-Webseite, das unten beschrieben wird](img/github-actions-workflow-dispatch.png "Bestätigungsmenü des Workflows im Tab GitHub Actions")
```

Comment:
Maybe

```suggestion
   ![Screenshot des Formulars auf der GitHub-Webseite, das unten beschrieben wird](img/github-actions-workflow-dispatch.png "Workflowmenü der GitHub-Action")
```

Reply:
It feels weird to describe an image as a "screenshot of" something instead of just describing the thing. But I guess let's just keep it in line with the English version.

Reply:
For people with no visual deficiencies, I guess you could say that it's redundant. Keep in mind though that this text is not shown on the normal website and is meant for screen readers.

Reply:
I did keep that in mind; alt text doesn't usually use phrasing like "an image of ..." and instead just describes the content of an image

Reply:
Yeah, let's follow the I would say styling choice here 🤷